### PR TITLE
Update removal message after convert

### LINKED ISSUE #49: Unwanted behaviour when cleanup performed not using plugin
Ok so I converted a cert-manager release from vv2 to v3:

```
❯ helm 2to3 convert --tiller-ns tiller cert-manager

Release "cert-manager" will be converted from Helm 2 to Helm 3.
[Helm 3] Release "cert-manager" will be created.
[Helm 3] ReleaseVersion "cert-manager.v1" will be created.
[Helm 3] ReleaseVersion "cert-manager.v1" created.
[Helm 3] Release "cert-manager" created.
Release "cert-manager" was converted successfully from Helm 2 to Helm 3. Note: the v2 releases still remain and should be removed to avoid conflicts with the migrated v3 releases.
```

Release existed in v2 & v3 as expected:

```
# helm2
❯ helm ls

NAME            REVISION    UPDATED                     STATUS      CHART                    APP VERSION    NAMESPACE
anchore         7           Tue Apr 16 13:12:03 2019    DEPLOYED    anchore-engine-0.12.1    0.3.3          default
cert-manager    1           Mon Apr 15 17:16:11 2019    DEPLOYED    cert-manager-v0.7.0      v0.7.0         cert-manager

#helm3
❯ helm3 ls -n cert-manager

NAME            NAMESPACE       REVISION    UPDATED                                    STATUS      CHART
cert-manager    cert-manager    1           2019-04-15 16:16:11.590230902 +0000 UTC    deployed    cert-manager-v0.7.0
```

So I attempted to delete the v2 release hoping that it would go from v2, remain in v3 and the pod would survive:

```
❯ helm delete cert-manager

Error: deletion completed with 1 error(s): release "cert-manager": object "" not found, skipping delete

❯ helm ls                                                                                                                  
NAME       REVISION    UPDATED                     STATUS      CHART                    APP VERSION    NAMESPACE
anchore    7           Tue Apr 16 13:12:03 2019    DEPLOYED    anchore-engine-0.12.1    0.3.3          default

❯ helm3 ls -n cert-manager

NAME            NAMESPACE       REVISION    UPDATED                                    STATUS      CHART
cert-manager    cert-manager    1           2019-04-15 16:16:11.590230902 +0000 UTC    deployed    cert-manager-v0.7.0
```

But unfortunately the actual pods were gone:

```
❯ kgp -n cert-manager

No resources found.
```

So I had the v3 release but no pods living under it (in any namespace).

## Sort of fix..

As I wasn't so bothered and didn't have much custom config, I decided to upgrade the v3 release. I had to upgrade the chart from v0.7.0 to v0.10.0 as before I did that I got this:

```
❯ helm3 upgrade cert-manager ./jetstack-cert-manager -n cert-manager
Error: validation: chart.metadata is required
```

On that basis I assumed that the older version of the chart did not meet the new requirements for helmv3.

For anyone who has similar this is my upgrade path:
```
helm3 search hub cert-manager
helm3 pull jetstack/cert-manager
helm3 upgrade cert-manager jetstack/cert-manager -n cert-manager --dry-run --debug

# actul upgrade
❯ helm3 upgrade cert-manager jetstack/cert-manager -n cert-manager

Release "cert-manager" has been upgraded. Happy Helming!
NAME: cert-manager
LAST DEPLOYED: 2019-09-27 15:49:02.86515 +0100 BST
NAMESPACE: cert-manager
STATUS: deployed
NOTES:
cert-manager has been deployed successfully!

In order to begin issuing certificates, you will need to set up a ClusterIssuer
or Issuer resource (for example, by creating a 'letsencrypt-staging' issuer).

More information on the different types of issuers and how to configure them
can be found in our documentation:

https://docs.cert-manager.io/en/latest/reference/issuers.html

For information on how to configure cert-manager to automatically provision
Certificates for Ingress resources, take a look at the `ingress-shim`
documentation:

https://docs.cert-manager.io/en/latest/reference/ingress-shim.html

# check release
helm3 ls -n cert-manager

NAME        	NAMESPACE   	REVISION	UPDATED                            	STATUS  	CHART
cert-manager	cert-manager	2       	2019-09-27 15:49:02.86515 +0100 BST	deployed	cert-manager-v0.10.0
```

## Question

Once I've used `helm 2to3 convert ...` and have a release in both v2 & v3 - should I be able to delete the v2 release? Should the pods transfer ownership to the v3 release?

Or does that only happen when I use the `--delete-v2-releases` flag?

Finally note my delete command errored but still seemed to delete the release which was confusing. Any advice appreciated.

### PR DESCRIPTION
**Pull request by rimusz**
Update removal message after a convert for better clarification on best approach.

Partial-Fix #49

### CODE REVIEW

File: cmd/convert.go [hunk 1]
```diff
@@ -123,7 +123,7 @@ func Convert(releaseName string) error {
 		}
 	} else {
 		if !settings.dryRun {
-			fmt.Printf("Release \"%s\" was converted successfully from Helm 2 to Helm 3. Note: the v2 releases still remain and should be removed to avoid conflicts with the migrated v3 releases.\n", releaseName)
+			fmt.Printf("Release \"%s\" was converted successfully from Helm 2 to Helm 3. Note: the v2 release information still remains and should be removed to avoid conflicts with the migrated v3 release.\n", releaseName)
```
Comment (hickeyma) [inline cmd/convert.go]: Can we add a sentence at the end of this message to tell the user how best to remove the v2 release info? Maybe something like this: 'v2 release information should be only be removed using `helm 2to3 cleanup` and when all releases have been migrated over.'

Comment (rimusz) [reply cmd/convert.go]: done

File: cmd/convert.go [hunk 2]
```diff
@@ -119,11 +119,13 @@ func Convert(releaseName string) error {
...
 
-			fmt.Printf("Release \"%s\" was converted successfully from Helm 2 to Helm 3.\n", releaseName)
+			fmt.Printf("Release \"%s\" was converted successfully from Helm v2 to Helm v3.\n", releaseName)
 		}
 	} else {
 		if !settings.dryRun {
-			fmt.Printf("Release \"%s\" was converted successfully from Helm 2 to Helm 3. Note: the v2 releases still remain and should be removed to avoid conflicts with the migrated v3 releases.\n", releaseName)
+			fmt.Printf("Release \"%s\" was converted successfully from Helm v2 to Helm v3.\n", releaseName)
+			fmt.Printf("Note: The v2 release information still remains and should be removed to avoid conflicts with the migrated v3 release.\n")
+			fmt.Printf("v2 release information should be only be removed using 'helm 2to3 cleanup' and when all releases have been migrated over.\n")
```
Comment (hickeyma) [inline cmd/convert.go]: Small change: "v2 release information should be only be removed using 'helm 2to3 cleanup' and when all releases have been migrated over.\n" --> "v2 release information should only be removed using `helm 2to3 cleanup` and when all releases have been migrated over.\n"

Comment (rimusz) [reply cmd/convert.go]: done

File: cmd/convert.go [hunk 3]
```diff
@@ -119,11 +119,13 @@ func Convert(releaseName string) error {
...
 
-			fmt.Printf("Release \"%s\" was converted successfully from Helm 2 to Helm 3.\n", releaseName)
+			fmt.Printf("Release \"%s\" was converted successfully from Helm v2 to Helm v3.\n", releaseName)
 		}
 	} else {
 		if !settings.dryRun {
-			fmt.Printf("Release \"%s\" was converted successfully from Helm 2 to Helm 3. Note: the v2 releases still remain and should be removed to avoid conflicts with the migrated v3 releases.\n", releaseName)
+			fmt.Printf("Release \"%s\" was converted successfully from Helm v2 to Helm v3.\n", releaseName)
+			fmt.Printf("Note: The v2 release information still remains and should be removed to avoid conflicts with the migrated v3 release.\n")
+			fmt.Printf("v2 release information should only be removed using `helm 2to3 cleanup` and when all releases have been migrated over.\n.\n")
```
Comment (hickeyma) [inline cmd/convert.go]: Small nit, extra full stop and newline at end: "v2 release information should only be removed using `helm 2to3 cleanup` and when all releases have been migrated over.\n.\n" --> "v2 release information should only be removed using `helm 2to3 cleanup` and when all releases have been migrated over.\n"

Comment (rimusz) [reply cmd/convert.go]: fixed, damn that copy-pasta :)

**Pull request merged**